### DISCUSSION TITLE
The columnlist comprises two columns with just some paragraphs in each of them. How to define it and append  it to a page.

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [jheddings/notional#166] [Q&A]
# The columnlist comprises two columns with just some paragraphs in each of them. How to define it and append  it to a page.

I read the documentation of notional. Still cant figure out how to construct nested_data for columnlist. Please help, thanks!

**Comment by jheddings** [comment]
Ugh, this one is nasty.  It's another example where the Notion API is inconsistent (like constructing tables).

Let me see if I can get an example working with some new helpers for columns.

**Reply by jheddings** [reply]
I've started PR #168 - feel free to take a look and comment.

If those example look reasonable, I'll get this out in the next release.

**Reply by tyehuoPHD** [reply]
thanks! I'll take look at the pr.

**Reply by jheddings** [reply]
FYI - I've gone ahead and merged the PR.  it is available in notional-0.8.0 if you want to try these new helpers.

**Reply by tyehuoPHD** [reply]
ok sir !
